src/pages/components/Dialog.js

```diff
@@ -35,7 +37,12 @@ export default function Dialog({ open, setOpen, feature }) {
 
   return (
     <Wrapper>
-      <DialogComponent width="60ch" ref={dialogRef} includeClose={true}>
+      <DialogComponent
+        width="60ch"
+        maxHeight="80ch"
```

Comment:
makes scroll bars appear if dialog content is long enough

Reply:
`ch` is fine. Note you can also do `lh` for line height that might be more intuitive

Reply:
that does make more sense! I'll change to `lh` in my next `hii-client` PR